MVCDemos/Views/Toolbar/Adaptability.cshtml

```diff
@@ -324,23 +354,31 @@
         DevExpress.ui.notify('The "Redo" button was clicked');
     }
 
-    function attachButtonClick() {
-        DevExpress.ui.notify('The "Attach" button was clicked');
+    function onLinkButtonClick() {
+        DevExpress.ui.notify('The "Link" button was clicked');
     }
 
-    function onAddButtonClick() {
-        DevExpress.ui.notify('The "Add" button was clicked');
+    function onAddImageButtonClick() {
+        DevExpress.ui.notify('The "Add Image" button was clicked');
     }
 
-    function onRemoveButtonClick() {
-        DevExpress.ui.notify('The "Remove" button was clicked');
+    function onClearButtonClick() {
+        DevExpress.ui.notify('The "Clear Formating" button was clicked');
     }
 
-    function onAboutButtonClick() {
-        DevExpress.ui.notify('The "About" button was clicked');
+    function onCodeBlockButtonClick() {
+        DevExpress.ui.notify('The "Code Block" button was clicked');
     }
 
-    function onCheckBoxValueChanged() {
-        DevExpress.ui.notify('The "Navigation Pane" checkbox value was changed');
+    function onQuoteButtonClick() {
+        DevExpress.ui.notify('The "Blockquote" button was clicked');
+    }
+
+    function attachButtonClick() {
```

Comment:
Please, ensure that naming is consistent everywhere

```suggestion
    function onAttachButtonClick() {
```

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2135/commits/86fbd66c983db71118dea44d78073a2f93d9975c